.github/workflows/main.yml

```diff
@@ -1,42 +1,188 @@
-name: Deploy GH Pages
-
-on:
-  push:
-    branches: 
-      - master 
-    paths:
-      - 'docs/*.md'
-      - 'docs/quickstart'
-      - '*.md'
-
-jobs:
-  deploy-gh-pages:
-    runs-on: ubuntu-latest
-    steps:
-      - name: Checkout master
-        uses: actions/checkout@v2
-        with:
-          path: master
-
-      - name: Checkout gh-pages branch
-        uses: actions/checkout@v2
-        with:
-          ref: 'gh-pages'
-          path: gh-pages
-    
-      - name: Copy site files
-        run: |
-          cp master/docs/*.md gh-pages/
-          cp -r master/docs/quickstart gh-pages/_pages
-          cp master/CODE_OF_CONDUCT.md master/CONTRIBUTING.md gh-pages/_pages/
-
-      - name: Commit GH Pages
-        run: |
-          cd gh-pages
-          git config user.name armada-admin
-          git config user.email admin@armadaproject.io
-          git add . _pages
-          git diff --quiet && git diff --staged --quiet || git commit -m "Updating Github Pages branch with latest Master changes"
-          git push
-        env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+# Pre-merge TODO list
+# 1. change DOCKERTOKEN and DOCKERUSERNAME
+name: CI
+
+defaults:
+  run:
+    shell: bash
+
+env:
+  GO111MODULE: "on"
+  GOPATH: "/home/runner/work/armada/armada/go"
+  GOCACHE: "/home/runner/.cache/go-build"
+
+on: 
+  push:
+    branches:
+      - '**'
+    pull_request:
+      branches:
+        - master
+    tags-ignore:
+      - '**'
+    paths-ignore:
+      - 'docs/*.md'
+      - 'docs/quickstart'
+      - '*.md'
+      - '**/release.yml'
+      - '**/pages.yml'
+
+jobs:
+  code_styles:

```

Comment:
Where do you do the equivalent of:
https://github.com/G-Research/armada/blob/master/.circleci/config.yml#L128

Checking typescript formatting

Reply:
Thanks! @JamesMurkin 
Added on [L44](https://github.com/G-Research/armada/pull/597/files#diff-7829468e86c1cc5d5133195b5cb48e1ff6c75e3e9203777f6b2e379d9e4882b3R44)